packages/primitives/popover/src/popover-arrow.directive.ts

```diff
@@ -95,9 +95,9 @@ export class RdxPopoverArrowDirective implements AfterViewInit {
         );
 
         this.renderer.setStyle(this.elementRef.nativeElement, 'top', posParams.top);
-        this.renderer.setStyle(this.elementRef.nativeElement, 'bottom', posParams.bottom);
+        this.renderer.setStyle(this.elementRef.nativeElement, 'bottom', '');
```

Comment:
```suggestion
```
or is it overwritten here?

Reply:
yes, leftover, it was filled with a value before. Will remove. Thanks.

Reply:
now I use only top ad left. Right and bottom are set to empty string to make sure they have no value e.g. from outside